Documentation/gettingstarted/kind.rst

```diff
@@ -48,36 +48,30 @@ Then, install Cilium release via Helm:
 
 .. note::
 
-   To enable Cilium's kube-proxy replacement (:ref:`kubeproxy-free`), kind nodes
-   need to have their own cgroup namespaces, and are different from the cgroup namespace
+   To enable Cilium's Socket LB (:ref:`kubeproxy-free`), Kind nodes
+   need to run in separate `cgroup namespaces
+   <https://man7.org/linux/man-pages/man7/cgroup_namespaces.7.html>`__,
+   and they need to be different from the cgroup namespace
    of the underlying host so that Cilium can attach BPF programs at the right
-   cgroup hierarchy. To verify this, run the following commands inside
-   respective kind nodes, and ensure that the cgroup values are different.
+   cgroup hierarchy. To verify this, run the following commands, and ensure
+   that the cgroup values are different:
 
    .. code-block:: shell-session
 
-      $ docker exec -it kind-control-plane bash
-      $ ls -al /proc/self/ns/cgroup
+      $ docker exec kind-control-plane ls -al /proc/self/ns/cgroup
       lrwxrwxrwx 1 root root 0 Jul 20 19:20 /proc/self/ns/cgroup -> 'cgroup:[4026532461]'
 
-      $ docker exec -it kind-worker bash
-      $ ls -al /proc/self/ns/cgroup
+      $ docker exec kind-worker ls -al /proc/self/ns/cgroup
       lrwxrwxrwx 1 root root 0 Jul 20 19:20 /proc/self/ns/cgroup -> 'cgroup:[4026532543]'
 
-   If you've deployed a kind cluster on an environment where kind nodes don't
-   have their own cgroup namespaces, then cgroup v2 needs to be enabled by
-   setting the kernel ``systemd.unified_cgroup_hierarchy=1`` parameter.
-   Also, cgroup v1 controllers ``net_cls`` and ``net_prio`` have to be disabled, or
-   cgroup v1 has to be disabled (e.g. by setting the kernel ``cgroup_no_v1="all"`` parameter)
-   on older kernels (< 5.14). This is the `kernel fix <https://github.com/torvalds/linux/commit/8520e224f547cd070c7c8f97b1fc6d58cff7ccaa>`__
-   that's available since kernel 5.14.
-   To verify this, run the following commands on the host, and check that the
-   output values are different.
-
-   .. code-block:: shell-session
+      $ ls -al /proc/self/ns/cgroup
+      lrwxrwxrwx 1 root root 0 Jul 19 09:38 /proc/self/ns/cgroup -> 'cgroup:[4026531835]'
 
-      $ sudo ls -al /proc/$(docker inspect -f '{{.State.Pid}}' kind-control-plane)/ns/cgroup
-      $ sudo ls -al /proc/self/ns/cgroup
+   Another requirement for the Socket LB on Kind to properly function is that either cgroup v1
+   controllers ``net_cls`` and ``net_prio`` are disabled (or cgroup v1 all together is disabled
+   e.g., by setting the kernel parameter ``cgroup_no_v1="all"``), or the host kernel
+   should be 5.14 or more recent to include the `fix
+   <https://github.com/torvalds/linux/commit/8520e224f547cd070c7c8f97b1fc6d58cff7ccaa>`__.
```

Comment:
“the fix.” sounds very mysterious. Which fix? I'd say “this fix” (implying the one we link), or “the required fix” maybe.